packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -338,6 +338,19 @@ export class PlaygroundWorkerEndpoint extends PHPWorker {
 			const primaryPhp = await requestHandler.getPrimaryPhp();
 			await this.setPrimaryPHP(primaryPhp);
 
+			await primaryPhp.writeFile(
+				'/wordpress/nonce-test.php',
+				`<?php
+					require_once '/wordpress/wp-load.php';
+					error_log(print_r($_COOKIE, true));
+					if (!empty($_COOKIE)) {
+						echo array_filter(array_keys($_COOKIE), function($key) {
+							return strpos($key, 'wordpress_logged_in_') === 0;
+						}) ? '1' : '0';
+					}
+				`
+			);
```

Comment:
@bgrgicak what's the purpose of this change?

Reply:
Thanks for catching this, I missed it. https://github.com/WordPress/wordpress-playground/pull/1928 will remove this code.